hearth/spec/hearth/http/field_spec.rb

```diff
@@ -42,22 +42,6 @@ module HTTP
           end
         end
 
-        context 'value is an Array' do
```

Comment:
`context 'value is a Scalar type'` can be demoted (since value will only ever be scalar type, no context necessary)